.github/workflows/labeler.yml

```diff
@@ -0,0 +1,24 @@
+name: "Label PR"
+
+on:
+  pull_request_target:
```

Comment:
nitpick
```suggestion
  - pull_request_target
```

Reply:
it's not equivalent is it ? from a dict to a list. Wouldn't it be better to remove it altogether since it doesn't seem used